conduit/plugins/importers/algod/algod_importer.go

```diff
@@ -154,6 +155,18 @@ func (algodImp *algodImporter) Close() error {
 	return nil
 }
 
+func (algodImp *algodImporter) getDelta(rnd uint64) (sdk.LedgerStateDelta, error) {
+	var delta sdk.LedgerStateDelta
+	params := algod.GetLedgerStateDeltaParams{Format: "msgp"}
+	data, err := (*common.Client)(algodImp.aclient).GetRaw(algodImp.ctx, fmt.Sprintf("/v2/deltas/%d", rnd), params, nil)
```

Comment:
We'll need to avoid this type after the SDK is updated. Not 100% sure how the requests are made, maybe this works?
```suggestion
	data, err := (*common.Client)(algodImp.aclient).GetRaw(algodImp.ctx, fmt.Sprintf("/v2/deltas/%d?format=msgp", rnd), params, nil)
```

Reply:
I guess I'll wait until the sdk changes are done and then update.